sbol_utilities/component.py

```diff
@@ -624,6 +645,48 @@ def part_in_backbone(identity: str, part: sbol3.Component, backbone: sbol3.Compo
     part_in_backbone_component.types.append(topology_type)
     return part_in_backbone_component, part_in_backbone_seq
 
+def part_in_backbone2(identity: str,  sequence: str, part_location: List[int], part_roles:List[str], fusion_site_length:int, linear:bool, **kwargs) -> Tuple[sbol3.Component, sbol3.Sequence]:
```

Comment:
Per our discussion, I don't believe this function makes sense in its current form.

Reply:
replaced with part_in_backbone_from_sbol